app/discover/procurement/page.tsx

```diff
@@ -0,0 +1,361 @@
+</Text>
+
+<Text>
+The changes into Oskari should never be done into the core of Oskari but as a separate whole. This makes the updating of the Oskari core much easier in the future.
+</Text>
+
+<Text>
+And to make sure that Oskari is a living, developing open source project for years to come for your organisation and everybody else, <b>it is highly recommended that you also contribute to the main Oskari GitHub branch and inform other Oskari users about what you are doing or have done.</b> This way other Oskari users can both enjoy the new functionalities and possibilities of Oskari but also contribute to the further development of the software. Oskari is open source at heart and if you create something Oskari-related, please publish it as open source.
+</Text>
+
+<h3>Documentation</h3>
+
+<Text>
+To make sure that the next developer who either updates your Oskari instance or further develops your new functionalities/configurations knows what to do and what to take into account, you must request proper documentation. 
+</Text>
+
+<Text>
+The documentation should include, for example, what kind of configurations have been done, how, and where. For new functionalities the names of the new bundles (basically modules of Oskari that do the wanted functionality), their events and requests should be listed. The documentation should be so comprehensive that a person who hasn’t been in the original development team can continue the work.
+</Text>
+
+<Text>
+The documentation can be requested as a separate document or as a Readme file that is included in the GitHub repository of your Oskari instance.
+</Text>
+
+<h3>Support and training</h3>
+
+<Text>
+After your new Oskari service is up and running, you or the people in your organisation might also need support and training. Easiest way to get support might be to join the Oskari Joint development forum and get to know other Oskari main users and organisations, their use cases, problems and wishes. 
+</Text>
+
+<Text>
+You can also procure support and training for Oskari and the software related to it from commercial partners. In some other procurement cases the experience with Oskari is not necessarily required from the vendor, but with support and training it is highly required.
+</Text>
+
+<h3>When can you use your own organisation&apos;s IT capabilities?</h3>
+
+<Text>
+If you have an in-house IT department, there are certain Oskari processes that can be done by them. These are also some of the easier things to do with Oskari that don’t require Oskari specific knowledge or expertise. The Oskari documentation is also being improved so that as many people as possible could use Oskari and join in the development.
```

Comment:
"Oskari processes"? If you have developers that know JS and Java, they might be very capable of installing and customizing Oskari. I think it's mostly about what capabilities the in-house IT has? It's not any harder or easier for an in-house developers to do than some other developer :)